.github/workflows/cwlcheck.yml

```diff
@@ -0,0 +1,36 @@
+name: cwlcheck
+on:
+  pull_request:
+    branches:
+      - develop
+  push:
+    branches:
+      - develop
+
+jobs:
+  build:
+    name: cwlcheck
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          path: repo
+
+      - uses: actions/checkout@v4
+        with:
+          repository: common-workflow-language/cwl-v1.2.git
+          path: cwl
+          ref: main
+
+
+      - name: install schema_salad and cwltool
+        run: |
+          pip install 'schema_salad @ git+https://github.com/common-workflow-language/schema_salad.git'
```

Comment:
`pip install schema-salad` should be good enough; I don't see any [unreleased commits](https://github.com/common-workflow-language/schema_salad/compare/8.4.20230927144413...main) that are required